build.py

```diff
@@ -0,0 +1,73 @@
+"""
+Copyright 2018 JanusGraph Python Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+__author__ = "Debasish Kanhar (https://github.com/debasishdebs)"
+__credits__ = ["Jason Plurad", "sjudeng", "Florian Hockman"]
```

Comment:
Not that important, but please add a second `n` to my surname: _Hockman_ -> _Hockmann_.

Apart from that, I'm not that familiar with bigger Python projects, but is it normal to have such a header in every file? Does having a `__version__` file in every file mean that we need to bump it every time we change something in that file? And wouldn't the git history already include the same information?
Or regarding the __email__ field: Will every contributor add his/her email address to that field? And again: Isn't that information already available via the git history?

Reply:
Well, I was thinking the same thing. Well `__version__` meant the version of library, but since that is already captured in `build.py` I think we should skip that. Maintaining that would be nighmare in future.
As for `__email__` well it's the email of author. Contributors/Reviewers are added under `__credits__`.

What I think will be cleaner is that, I just keep License text, author, credits and email and remove everything else. What would you say?

Reply:
I personally don't like headers like that at all as they are hard to maintain and most of the information is already available elsewhere. That you're the author can for example also be seen in the git history and also in the `AUTHORS.txt`. @mbrukman made [a comment that goes in the same direction](https://github.com/JanusGraph/janusgraph-python/pull/4#discussion_r215828794).